### DISCUSSION TITLE
Is it possible to send Multi-Product Messages?

### DISCUSSION THREAD

**Discussion by DeiverJC** [netflie/whatsapp-cloud-api#96] [Q&A] [answered]
Hello everyone,

Does the package allow sending Multi-Product Messages? I'm looking at the documentation and I haven't seen any reference to this type of message.

Thank you very much in advance for the package, it is an exceptional job.

Greetings!

**Comment by aalbarca** [comment, marked as answer]
Unfortunately I have not implemented this functionality.

Would you like to open a PR with the functionality implemented? Here is an example: https://developers.facebook.com/docs/whatsapp/cloud-api/reference/messages

**Reply by DeiverJC** [reply]
Ok, I would like to make that contribution to the package.
